ee/modules/610-service-with-healthchecks/crds/doc-ru-servicewithhealthchecks.yaml

```
@@ -46,81 +44,80 @@ spec:
+                     Если IP-адрес указывается вручную, он должен находиться в пределах допустимого диапазона (заданного системной конфигурацией) и быть свободным для назначения; в противном случае создание сервиса завершится ошибкой.
+                     Это поле не подлежит изменению при обновлениях, за исключением случаев, когда тип сервиса также изменяется на `ExternalName` (тогда это поле должно оставаться пустым) или если тип меняется с `ExternalName`, позволяя при необходимости указать это поле, как указано выше. 
                      
                      Допустимые значения: `None`, пустая строка (""), или действительный IP-адрес. Установка этого значения в `None` создает сервис без виртуального IP, что полезно, когда предпочтительнее прямое подключение к конечным точкам,
-                     и проксирование не требуется. Это применяется только к типам ClusterIP, NodePort и LoadBalancer. Если это поле указано при создании сервиса типа ExternalName, создание завершится неудачей. 
+                     и проксирование не требуется. Применимо только к типам ClusterIP, NodePort и LoadBalancer. Если это поле указано при создании сервиса типа `ExternalName`, создание завершится неудачей. 
                      
                      Это поле будет очищено при обновлении сервиса на тип ExternalName. [Подробнее.](https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)
                 clusterIPs:
                   description: |-
-                    ClusterIPs — это список IP-адресов, назначенных сервису, которые обычно назначаются случайным образом.
-                    Если адрес указан вручную, он должен быть в пределах допустимого диапазона (в соответствии с конфигурацией системы) и не быть занятым,
-                    чтобы быть выделенным для сервиса; в противном случае создание сервиса завершится неудачей. Это поле нельзя изменять через обновления,
-                    если поле типа также не изменяется на ExternalName (что требует, чтобы это поле было пустым) или если поле типа изменяется с ExternalName
-                    (в этом случае это поле может быть указано дополнительно, как описано выше). Допустимые значения: `None`, пустая строка (""), или действительный IP-адрес.
+                    `ClusterIPs` — это список IP-адресов, назначенных сервису, обычно выделяемых случайным образом.
+                    Если IP-адрес указывается вручную, он должен находиться в допустимом диапазоне (заданном конфигурацией системы) и быть свободным для назначения;
+                    в противном случае создание сервиса завершится ошибкой. Поле не подлежит изменению при обновлении, за исключением случаев, когда тип сервиса также изменяется на `ExternalName` (в этом случае поле должно быть пустым)
+                    или когда тип меняется с `ExternalName`, позволяя задать это поле дополнительно, как указано выше. 
```

```suggestion
                    Список IP-адресов, назначенных сервису.
                    
                    Обычно, IP-адреса назначются сервису случайным образом. Если IP-адрес указывается вручную, он должен находиться в допустимом диапазоне (заданном конфигурацией системы) и быть свободным для назначения; в противном случае создание сервиса завершится ошибкой.
                    
                    Поле не подлежит изменению при обновлении, за исключением случаев, когда тип сервиса также изменяется на `ExternalName` (в этом случае поле должно быть пустым), или когда тип меняется с `ExternalName`, позволяя задать это поле дополнительно, как указано выше. 
```

Чет сложно-длинно. Но пусть

---

В этом параметре прям много текст появилось. Надо бы отрендерить и глянуть как тут будет в итоге.